server/py/services/api/tests/unit/db/test_model_endpoint.py

```diff
@@ -160,95 +177,85 @@ def test_list_filters(self) -> None:
                 "label2": f"value_{i+1}",
                 "label": "value",
             }
-            uid = self._db.store_model_endpoint(
+            mep = self._db.store_model_endpoint(
                 self._db_session,
                 model_endpoint,
                 name=model_endpoint.metadata.name,
                 project=model_endpoint.metadata.project,
+                function_name="function-1",
             )
-            uids.append(uid)
+            uids.append(mep.metadata.uid)
 
         list_mep = self._db.list_model_endpoints(
             self._db_session,
             project=model_endpoint.metadata.project,
             model_name="model-1",
-        )
+        ).endpoints
         assert len(list_mep) == 2
 
         list_mep = self._db.list_model_endpoints(
             self._db_session,
             project=model_endpoint.metadata.project,
             model_name="model-2",
-        )
+        ).endpoints
         assert len(list_mep) == 0
 
         list_mep = self._db.list_model_endpoints(
             self._db_session, project=model_endpoint.metadata.project, latest_only=True
-        )
+        ).endpoints
         assert len(list_mep) == 1
 
         list_mep = self._db.list_model_endpoints(
             self._db_session,
             project=model_endpoint.metadata.project,
             labels=["label=value"],
-        )
+        ).endpoints
         assert len(list_mep) == 2
 
         list_mep = self._db.list_model_endpoints(
             self._db_session,
             project=model_endpoint.metadata.project,
             labels=["label1=value_0"],
-        )
+        ).endpoints
         assert len(list_mep) == 1
 
         list_mep = self._db.list_model_endpoints(
             self._db_session, project=model_endpoint.metadata.project, uids=uids
-        )
+        ).endpoints
         assert len(list_mep) == 2
 
         list_mep = self._db.list_model_endpoints(
             self._db_session, project=model_endpoint.metadata.project, uids=["uids"]
-        )
+        ).endpoints
         assert len(list_mep) == 0
 
         list_mep = self._db.list_model_endpoints(
             self._db_session,
             project=model_endpoint.metadata.project,
             function_name="function-1",
-        )
+        ).endpoints
         assert len(list_mep) == 2
 
-        mep = self._db.get_model_endpoint(
-            self._db_session,
-            project=model_endpoint.metadata.project,
-            name="model-endpoint-1",
-            uid=uids[1],
-        )
-        list_mep = self._db.list_model_endpoints(
-            self._db_session,
-            project=model_endpoint.metadata.project,
-            start=mep.metadata.created,
-        )
-        assert len(list_mep) == 1
-
-        model_endpoint.metadata.endpoint_type = 3
```

Comment:
Why was it removed?

Reply:
This question remained without a clear answer. It's not critical, but please update for the reason